packages/knip/src/util/tag.ts

```diff
@@ -6,7 +6,7 @@ export const splitTags = (tags: string[]) =>
     .flatMap(tag => tag.split(','))
     .reduce<Tags>(
       ([incl, excl], tag) => {
-        (tag.startsWith('-') ? excl : incl).push(tag.match(/[a-zA-Z]+/)![0]);
+        (tag.startsWith('-') ? excl : incl).push(/[a-zA-Z]+/.exec(tag)![0]);
```

Comment:
If we are really itching for performance, RegEx is very slow: https://jsperf.app/tezuge. That might be premature optimization though ahah.

Reply:
I don't think `String.includes()` works with Regex.